warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PushdownLowSelectivityNodesVisitor.java

```diff
@@ -58,7 +59,10 @@ public Object visit(ASTReference node, Object data) {
     public Object visit(ASTEQNode node, Object data) {
         // if this node represents a field/value that has poor selectability, then push it down
         if (hasLowSelectability(node)) {
-            return ASTDelayedPredicate.create(node);
+            JexlNode parent = node.jjtGetParent();
+            JexlNode delayed = ASTDelayedPredicate.create(node);
```

Comment:
the create call should already be reparenting

Reply:
I'll update per our conversation